person.rb

```diff
@@ -27,6 +28,10 @@ def can_use_services?
   def of_age?
     @age >= 18
   end
+
+  def add_rental(person, date)
+    @rentals << Rental.new(date, self, person)
+  end
```

Comment:
- [ ] We should request book and date here, not person and date. And we should organize the parameters well.

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```